colossus-metrics/src/main/scala/colossus/metrics/collectors/Rate.scala

```diff
@@ -21,16 +21,13 @@ object Rate {
 
   def apply(address: MetricAddress, periods: List[FiniteDuration] = List(1.second, 60.seconds)): RateParams = RateParams(address, periods)
 
-  //this appears to be obselete now, AFAICT shared rates will only be created by called .shared on a local rate
-  implicit object SharedRate extends Generator[SharedLocality,Rate, RateParams] {
-    def apply(params: RateParams)(implicit collector: ActorRef): Shared[Rate] = {
+  implicit object LocalRate extends Generator[Rate, RateParams] {
```

Comment:
Is LocalRate a valid name for this Generator?  Seems like it generates both local and shared.  Should it just be RateGenerator, or something along those line?